test/runtests.jl

```diff
@@ -309,7 +309,7 @@ end
         return SA[x+x^3*κ^2/6,y-x^2*κ/2]
     end
     # Test sim_time, and sim_step! stopping time
-    sim = Simulation(radius.*(4,4),(1,0),radius; body=AutoBody(circle), ν, T)
+    sim = Simulation(radius.*(4,4),(1,0),radius; body=AutoBody(circle), ν, T, exitBC=true)
```

Comment:
I had to update this test again since it was failing locally on 1.10.0.

Reply:
This is the issue we're seeing in #98 again?

Reply:
Yes, exactly.